### PR TITLE
MCLOUD-5943: Remove root user

### PR DESCRIPTION
**Pull request by shiftedreality**
### Description

Replace the root user for PHP containers with correct `www:www` user

### Fixed Issues (if relevant)

1. https://jira.corp.magento.com/browse/MCLOUD-5943

### Manual testing scenarios

1. Build PHP and Nginx images
2. Test Developer mode
3. Test Production mode

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

**Comment by shiftedreality** [comment]
Travis is failing in mainline: https://github.com/magento/magento-cloud-docker/pull/219

**Review by BaDos — DISMISSED** [review]

**Review by oshmyheliuk — DISMISSED** [review]

**Comment by andriyShevtsov** [comment]
@shiftedreality I can't build php-cli image 7.3 locally:
```Step 24/40 : RUN docker-php-ext-enable   bcmath   blackfire   bz2   calendar   exif   gd   geoip   gettext   gmp   gnupg   igbinary   imagick   imap   intl   ldap   mailparse   msgpack   mysqli   oauth   opcache   pcov   pdo_mysql   propro   pspell   raphf   recode   redis   shmop   soap   sockets   sodium   sysvmsg   sysvsem   sysvshm   tidy   xdebug   xmlrpc   xsl   yaml   zip   pcntl   ioncube
 ---> Running in dc1f9c643dfc
error: 'oauth' does not exist

usage: /usr/local/bin/docker-php-ext-enable [options] module-name [module-name ...]
   ie: /usr/local/bin/docker-php-ext-enable gd mysqli
       /usr/local/bin/docker-php-ext-enable pdo pdo_mysql
       /usr/local/bin/docker-php-ext-enable --ini-name 0-apc.ini apcu apc

Possible values for module-name:
bcmath.so blackfire.so bz2.so calendar.so exif.so gd.so geoip.so gettext.so gmp.so gnupg.so igbinary.so imagick.so imap.so intl.so ioncube.so ldap.so mailparse.so msgpack.so mysqli.so opcache.so pcntl.so pcov.so pdo_mysql.so propro.so pspell.so raphf.so recode.so redis.so shmop.so soap.so sockets.so sodium.so sysvmsg.so sysvsem.so sysvshm.so tidy.so xdebug.so xmlrpc.so xsl.so yaml.so zip.so

Some of the above modules are already compiled into PHP; please check
the output of "php -i" to see which modules are already loaded.
The command '/bin/sh -c docker-php-ext-enable   bcmath   blackfire   bz2   calendar   exif   gd   geoip   gettext   gmp   gnupg   igbinary   imagick   imap   intl   ldap   mailparse   msgpack   mysqli   oauth   opcache   pcov   pdo_mysql   propro   pspell   raphf   recode   redis   shmop   soap   sockets   sodium   sysvmsg   sysvsem   sysvshm   tidy   xdebug   xmlrpc   xsl   yaml   zip   pcntl   ioncube' returned a non-zero code: 1
```

**Comment by andriyShevtsov** [comment]
I removed  `oauth` extension   from `dockerfile` and it works fine now but I’m getting next error when I’m trying to open magento  site 
```
500 Internal Server Error
nginx/1.9.15
```
@shiftedreality please take a look

**Comment by andriyShevtsov** [comment]
This issue is not related to current PR. I got the same error with another one PR.

File: images/nginx/1.9/Dockerfile [hunk 1]
```diff
@@ -30,8 +28,6 @@ COPY docker-entrypoint.sh /docker-entrypoint.sh
 RUN ["chmod", "+x", "/docker-entrypoint.sh"]
 ENTRYPOINT ["/docker-entrypoint.sh"]
 
-USER root
-
 EXPOSE 80
```
Comment (kandy) [inline images/nginx/1.9/Dockerfile:31]: You need root to use 80 port

Comment (kandy) [reply images/nginx/1.9/Dockerfile:31]: see https://github.com/nginxinc/docker-nginx-unprivileged

**Comment by andriyShevtsov** [comment]
root user need to open any port less then 1024

**Pull request closed without merging**